packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -173,9 +173,659 @@
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "key": {
+            "type": "string"
+          },
+          "value": {
+            "type": "string"
+          },
+          "oldValue": {
+            "type": "string"
+          },
+          "check_empty": {
+            "type": "boolean"
+          }
+        }
+      },
+      "GetAccountRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account.",
+        "type": "object",
+        "required": [
+          "accountId"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetBlockRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Block.",
+        "type": "object",
+        "required": [
+          "height"
+        ],
+        "properties": {
+          "height": {
+            "type": "integer"
+          }
+        }
+      },
+      "GetSignatoriesRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Signatories.",
+        "type": "object",
+        "required": [
+          "accountId"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Transactions.",
+        "type": "object",
+        "required": [
+          "txHashesList"
+        ],
+        "properties": {
+          "txHashesList": {
             "type": "array",
-            "items": {}
+            "items":{
+              "type": "string"
+            }
+          }
+        }
+      },
+      "GetPendingTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Pending Transactions.",
+        "type": "object",
+        "required": [
+          "pageSize"
+        ],
+        "properties": {
+          "pageSize": {
+            "type": "integer"
+          },
+          "firstTxHash": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Transactions.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "pageSize",
+          "firstTxHash"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "pageSize": {
+            "type": "integer"
+          },
+          "firstTxHash": {
+            "type": "string"
+          },
+          "firstTxTime": {
+            "type": "string"
+          },
+          "lastTxTime": {
+            "type": "string"
+          },
```

Comment:
I'm thinking about those types. According to iroha documentation they are timestamps: https://iroha.readthedocs.io/en/develop/develop/api/queries.html#result-pagination (Timestamps with milliseconds). So I don't know if it is good to use just `string`, or maybe `integer` or `date-time`.

We will see what @petermetz would decide.

Reply:
Sure, I will update it accordingly.

Reply:
> Sure, I will update it accordingly.

@yashrajdesai My answer is that whichever works (e.g. results in the correct automatic conversions) should be used. Right now, we don't know which one that would be though because the test case does not use these parameters at all, so the way to go is to add another test case within the existing test file where these parameters get used and then write an assertion to verify that the parameters are working as intended. If they don't, then we can tweak the types of the request properties until they do.